keps/sig-node/4004-deprecate-kube-proxy-version/kep.yaml

```diff
@@ -0,0 +1,42 @@
+title: Deprecate status.nodeInfo.kubeProxyVersion field
+kep-number: 4004
+authors:
+  - "@HirazawaUi"
+owning-sig: sig-network
+participating-sigs:
+  - sig-node
+status: provisional
+creation-date: 2023-05-15
+reviewers:
+  - "@danwinship"
+  - "@thockin"
+approvers:
+  - "@danwinship"
+
+see-also: []
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.28"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  alpha: "v1.28"
+  beta: "v1.29"
+  stable: "v1.30"
```

Comment:
we should be able to do alpha in 1.29, beta in 1.31 (3 releases after 1.28, to avoid version skew issues with cloud-provider-gcp), and stable in 1.33 (your graduation criteria say "no issues reported in 2 releases")

Reply:
fixed